src/components/incentives/MeritIncentivesTooltipContent.tsx

```diff
@@ -6,13 +6,17 @@ import { FormattedNumber } from '../primitives/FormattedNumber';
 import { Link } from '../primitives/Link';
 import { Row } from '../primitives/Row';
 import { TokenIcon } from '../primitives/TokenIcon';
+import { getSymbolMap } from './IncentivesTooltipContent';
 
 export const MeritIncentivesTooltipContent = ({
-  incentiveAPR,
-  rewardTokenSymbol,
-}: Omit<ReserveIncentiveResponse, 'rewardTokenAddress'>) => {
+  meritIncentives,
```

Comment:
what is this type?

Reply:
It's of type `ReserveIncentiveResponse` why?